Makefile

```diff
@@ -24,7 +24,7 @@ get-version:  ## Return version.
 
 .PHONY: dev
 dev: check-port install dev-env-up  ## Start development environment.
-	bash -c "trap 'make dev-env-down' EXIT; pnpm dev"
```

Comment:
only start hasura-auth, not graphql-codegen, that can be run manually when required